apps/builder/app/builder/features/workspace/canvas-iframe.tsx

```diff
@@ -23,7 +23,17 @@ export const CanvasIframe = forwardRef<HTMLIFrameElement, CanvasIframeProps>(
       $canvasIframeState.set("idle");
     });
 
-    return <iframe {...props} ref={ref} className={iframeStyle()} />;
+    const credentialless = {
+      credentialless: "true",
+    };
+    return (
+      <iframe
+        {...props}
+        ref={ref}
+        className={iframeStyle()}
+        {...credentialless}
```

Comment:
why not just prop?

Reply:
typescript error ;)

Reply:
decided to not extend interface

Reply:
would be more obvious to silence ts in that place or at least add a comment ...